src/get-rollup-options.js

```diff
@@ -5,10 +5,14 @@ import req from 'req-cwd'
 import stringifyAuthor from 'stringify-author'
 
 function getDest(options, format, compress) {
-  const filename = options.filename
+  let filename = options.filename
   const dir = options.outDir
   let suffix = ''
 
+  if (filename.startsWith('@')) {
```

Comment:
I think filename could contain `@` and does not have to be an scoped npm package name, like `@my-lib`. We may only need to update the logic of getting `filename` from `pkg.name`, I mean here https://github.com/egoist/bili/blob/master/src/get-config.js#L8